src/handlers/TechRecordsListHandler.ts

```diff
@@ -135,21 +142,38 @@ export class TechRecordsListHandler<T extends Vehicle> {
       );
     }
 
-    if (techRecordItem.techRecord.length <= 0) {
-      throw new HTTPError(404, HTTPRESPONSE.RESOURCE_NOT_FOUND);
-    }
-
     return techRecordItem;
   }
 
-  private formatTechRecordItemsForResponse(techRecordItems: T[]) {
-    const recordsToReturn = [];
-    return techRecordItems.map(this.formatTechRecordItemForResponse);
-    // for (let techRecordItem of techRecordItems) {
-    //   techRecordItem = this.formatTechRecordItemForResponse(techRecordItem);
-    //   recordsToReturn.push(techRecordItem);
-    // }
-    // return recordsToReturn;
+  private mergeRecordsWithSameSystemNumber(techRecordItems: T[]) {
+    const recordsToReturn: T[] = [];
+
+    techRecordItems.forEach((vehicle) => {
+      vehicle.techRecord.forEach((object, index) => {
+        vehicle.techRecord[index].historicVin = vehicle.vin;
```

Comment:
Should this be historic vin? At the time on the tech record, it isn't historic right? Like a tech record doesn't have history, so how can something be historic? Does that make some sense?

Reply:
I went about calling it historicVin because I was trying to say this is the vin of this tech record of this vehicle's point in history. I just didn't quite like the idea of having 2 attributes called vin in one response. I'm up for suggestions of what else it could be called?

Reply:
Like the vin at the based of the dynamo record is the vin right now at present.

Reply:
I think it should just be VIN personally as that's what it is. It's techRecord.vin. I also agree that sucks but the reason it sucks is the vehicle one should be called currentVin or something. I'll approve and you can ignore me tho haha